unstructured/embed/embedder/example.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from unstructured.embed.embedder.open_ai import OpenAIEmbedder, OpenAIEmbedderConfig
+
+config = OpenAIEmbedderConfig(
+    api_key=os.environ["OPENAI_API_KEY"],
+    list_of_elements_json_paths="""/Users/ahmet/elements_results/1.docx.json
+                                   /Users/ahmet/elements_results/2.pdf.json""",
```

Comment:
Don't want to include paths specific to the user.

Reply:
Updated with https://github.com/Unstructured-IO/unstructured/pull/1368/commits/d7a8689463f69a5591cc5bb6fbd2cabd84e60f96